jython/LoconetOpsModeSim.py

```diff
@@ -0,0 +1,108 @@
+# This script does a simple Digitrax SIM for Decoder Ops Mode  Read and Writes.
+# It can be useful when testing e.g. DecoderPro using a LocoNet simulator connection.
```

Comment:
I'm aware that it's difficult to move old scripts, but maybe it's good to put new LocoNet scripts in jython/loconet. We already have folders for LCC and MQTT for example.

Reply:
Good suggestion, thanks!  Done.